### ISSUE TITLE
Show the name of the stream in addition to the application name

### ISSUE THREAD

**Issue by 532910** [patroclos/PAmix#61]
Please add support to show track name, same as pavucontrol does.

**Comment by patroclos** [comment]
pavucontrol doesn't show tracknames.
maybe we should display the raw sinkinput name, since programs like mpv use that to show the filename

**Comment by 532910** [comment]
![image](https://user-images.githubusercontent.com/14617699/86542231-80da4880-bf1c-11ea-953c-322e41c6338f.png)

**Comment by patroclos** [comment]
Yeah, that's the name of the Pulseaudio stream, not the track and from the proplist. I remember experimenting around with this stuff and mpv doesn't seem to set the trackname (PA_PROP_MEDIA_TITLE), but just incorporates the filename in the stream name.
